packages/backend/src/openapi/webhooks.yaml

```diff
@@ -0,0 +1,275 @@
+          description: Return a 200 status to indicate that the data was received successfully
+  outgoingPaymentFailed:
+    post:
+      requestBody:
+        description: Notify account servicing entity that an outgoing payment failed and won't be retried. Funds should be withdrawn.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/outgoingPayment'
+      responses:
+        '200':
+          description: Return a 200 status to indicate that the data was received successfully
+  webMonetization:
+    post:
+      requestBody:
+        description: Notify account servicing entity that a Web Monetization payment was received and funds should be withdrawn.
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/webMonetization'
+      responses:
+        '200':
+          description: Return a 200 status to indicate that the data was received successfully
+
+components:
+  schemas:
+    incomingPayment:
+      required:
+        - id
+        - type
+        - attempts
+        - processAt
+        - data
+      properties:
+        id:
+          type: string
+          format: uuid
+        type:
+          type: string
+          enum:
+            - incoming_payment.completed
+            - incoming_payment.expired
+        attempts:
+          type: integer
+        processAt:
+          type: string
+          format: date-time
+        data:
+          type: object
+          required:
+            - id
+            - paymentPointerId
+            - createdAt
+            - updatedAt
+            - expiresAt
+            - receivedAmount
+            - completed
+          properties:
+            id:
+              type: string
+              format: uuid
+            paymentPointerId:
+              type: string
+              format: uuid
+            completed:
+              type: boolean
+            incomingAmount:
+              $ref: ./schemas.yaml#/components/schemas/amount
+            receivedAmount:
+              $ref: ./schemas.yaml#/components/schemas/amount
+            description:
+              type: string
+            externalRef:
+              type: string
+            createdAt:
+              type: string
+              format: date-time
+            updatedAt:
+              type: string
+              format: date-time
+            expiresAt:
+              type: string
+              format: date-time
+        statusCode:
+          type: integer
+        withdrawal:
+          $ref: '#/components/schemas/withdrawal'
+    outgoingPayment:
+      required:
+        - id
+        - type
+        - attempts
+        - processAt
+        - data
+      properties:
+        id:
+          type: string
+          format: uuid
+        type:
+          type: string
+          enum:
+            - outgoing_payment.created
+            - outgoing_payment.completed
+            - outgoing_payment.failed
```

Comment:
I wish these were just upper snake case eg `OUTGOING_PAYMENT_CREATED` 💅 , but for later